### PR TITLE
Environment_Engine: Remove calls to GetShallowClone

### LINKED ISSUE #2559: Environment_Engine: Replace calls to Clone methods with calls to Base DeepClone or ShallowClone
#### Broken rules:
See https://github.com/BHoM/admin/issues/9

#### Suggestions to restore compliance:
https://github.com/BHoM/admin/issues/9#issuecomment-709032648

### PR DESCRIPTION
**Pull request by alelom**
### Issues addressed by this PR

Closes #2559

### CODE REVIEW

**Review by FraserGreenroyd — APPROVED** [review]
LGTM

File: Environment_Engine/Modify/Transform.cs [hunk 1]
```diff
@@ -47,7 +48,7 @@ public static Space Transform(this Space space, TransformMatrix transform, doubl
                 return null;
             }
 
-            Space result = space.GetShallowClone() as Space;
+            Space result = space.ShallowClone() as Space;
```
Comment (FraserGreenroyd) [inline Environment_Engine/Modify/Transform.cs:51]:
```suggestion
            Space result = space.ShallowClone();
```

**Review by FraserGreenroyd — CHANGES_REQUESTED** [review]

File: Environment_Engine/Modify/Transform.cs (hunk 1 above)
Comment (FraserGreenroyd) [reply Environment_Engine/Modify/Transform.cs:51]: Is that better @alelom - will let you decide as you have the current spatial awareness of the situation.

Comment (alelom) [reply Environment_Engine/Modify/Transform.cs]: Yes sorry I realized you could do that only later

**Review by FraserGreenroyd — APPROVED** [review]

**Comment by FraserGreenroyd** [comment]
@BHoMBot check required
@BHoMBot check copyright-compliance
@BHoMBot check dataset-compliance
@BHoMBot check ready-to-merge

**Pull request merged**